api/graphql/schema.py

```diff
@@ -613,6 +613,18 @@ async def discussion(
         discussion = await loader.load(root.id)
         return GraphQLDiscussion.from_internal(discussion)
 
+    @strawberry.field()
+    async def sequencing_group_history(
+        self, info: Info[GraphQLContext, 'Query'], root: 'GraphQLProject'
+    ) -> list['GraphQLSequencingGroupTypeByDate']:
+        loader = info.context['loaders'][
```

Comment:
This needs a permission check before the data is loaded to ensure that the user has permission to view the project. 

```python
connection = info.context['connection']
connection.check_access_to_projects_for_ids([root.id], ReadAccessRoles)
```